commands/actions/common.go

```diff
@@ -170,6 +172,23 @@ func mustGetProjectActions(actions map[string]actionsModel.ProjectActions, proje
 	return &ret
 }
 
+func mustValidateDependencies(packageJSON *typescript.PackageJson, validator *packagejson.Validator) (*packagejson.ValidationResult, error) {
+	depResult, err := validator.Validate(packageJSON.Dependencies)
+	if err != nil {
+		return nil, err
+	}
+
+	devDepResult, err := validator.Validate(packageJSON.DevDependencies)
```

Comment:
Do we care about dev dependencies 🤔

Reply:
Well right now no, as we don't have any dev dependencies. But in the future if we add any dev dependencies to our runtime, this will be useful as we can prevent the overriding issues we've had with regular dependencies.

Reply:
Agreed. Let's keep it.